doc/setup/quickstart_cmake/index.md

```diff
@@ -237,9 +237,5 @@ target_link_libraries (my_app PUBLIC seqan3::seqan3)
   and run `cmake` again.
 
 * **SDSL library not found**: `The SDSL library is required, but wasn't found.`
-  Or similar for other dependencies. Dependencies are checked by compiling a small file. A library not being found can
-  have two potential causes:
-  - The repository was not cloned correctly. This can be verified by checking whether
-    `submodules/sdsl-lite/include/version.hpp` exists.
-  - The incorrect compiler is used (e.g., Apple Clang instead of GCC). Be sure to set `-DCMAKE_CXX_COMPILER=`. For an
-    example, see \ref remark_cmake_cxx_compiler "this remark".
+  The repository was not cloned correctly. This can be verified by checking whether
+  `submodules/sdsl-lite/include/version.hpp` exists.
```

Comment:
```suggestion
  The repository was not cloned correctly. This can be verified by checking whether
  `submodules/sdsl-lite/include/version.hpp` exists.
  If it doesn't, try this within the seqan3 directory:
  `git submodule update --init`
```

Reply:
https://github.com/seqan/seqan3/pull/3079